src/test/java/teammates/test/driver/EmailChecker.java

```diff
@@ -40,6 +48,25 @@ public static void verifyEmailContent(String emailContent, String filePathParam)
         }
     }
     
+    /**
+     * Verifies that given {@code emailContent} contains information about {@code inviter}.
+     * 
+     * @param inviter
+     * @param emailContent
+     * 
+     * @see EmailGenerator#generateInstructorCourseJoinEmail()
+     */
+    public static void verifyEmailContainsInviterInfo(AccountAttributes inviter, String emailContent) {
```

Comment:
So I need to totally remove this test? Or move it to EmailGeneratorTest file? Because it test ensures that generated email contains information about person who sends invite and checks that this enhancement works as expected?